doc/arkode/guide/source/Usage/ARKStep_c_interface/Relaxation.rst

```diff
@@ -0,0 +1,277 @@
+.. -----------------------------------------------------------------------------
+   Programmer(s): David J. Gardner @ LLNL
+   -----------------------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2022, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   -----------------------------------------------------------------------------
+
+.. _ARKODE.Usage.ARKStep.Relaxation:
+
+Relaxation Methods
+==================
+
+This section describes user-callable functions for utilizing relaxation methods
+with ARKStep. For more information on relaxation Runge-Kutta methods see
+:numref:`ARKODE.Mathematics.Relaxation`.
+
+Enabling or Disabling Relaxation
+--------------------------------
+
+.. c:function:: int ARKStepSetRelaxFn(void* arkode_mem, ARKRelaxFn rfn, ARKRelaxJacFn rjac)
+
+   Attaches the user supplied functions for evaluating the relaxation function
+   (``rfn``) and its Jacobian (``rjac``).
+
+   :param arkode_mem: the ARKStep memory structure
+   :param rfn: the user-defined function to compute the relaxation function
+               :math:`\xi(y)`
+   :param rjac: the user-defined function to compute the relaxation Jacobian
+                :math:`\xi'(y)`
+
+   :retval ARK_SUCCESS: the function exited successfully
+   :retval ARK_MEM_NULL: ``arkode_mem`` was ``NULL``
+   :retval ARK_ILL_INPUT: an invalid input combination was provided see the
+                          output error message for more details
+   :retval ARK_MEM_FAIL: a memory allocation failed
+
+   .. note::
+
+      If both ``rfn = rjac = NULL`` relaxation is disabled.
```

Comment:
If only one of the two is `NULL` an error is returned. When relaxation is combined with fixed time stepping ARKODE will always attempt the step with the fixed step size and then relax the step at the end of each step. The requirement of non-negative `b` values and an at least second order method are checked during initialization. I've updated the documentation to note this behavior.

Reply:
So if it ends up relaxing the result of a fixed step, it ends up using a different step size after all?  I anticipate that this may not be what users expect (if they have any "expectations" when running both relaxation RK and fixed time-stepping, or for that matter, if there are any users who would even contemplate both).  

Perhaps a comment/warning in this file that if relaxation is used along with fixed time-stepping, then obviously the integrator will not end up taking identical step sizes, although any implicit solves will?